From: Ryan Lee <ryan.lee@canonical.com>
Date: Tue, 31 Mar 2026 12:12:42 -0700
Subject: [PATCH] profiles: add xml, icc,
 rdf extensions to allowed ghostscript extensions

xml and icc are used for ZUGFeRD/Factur-X invoices, while rdf is used for
semantic PDF annotations.

Fixes: https://bugs.launchpad.net/bugs/2139339

Signed-off-by: Ryan Lee <ryan.lee@canonical.com>
---
 profiles/apparmor.d/tunables/gs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/profiles/apparmor.d/tunables/gs b/profiles/apparmor.d/tunables/gs
index 1db6262..8443c35 100644
--- a/profiles/apparmor.d/tunables/gs
+++ b/profiles/apparmor.d/tunables/gs
@@ -9,6 +9,6 @@
 #------------------------------------------------------------------
 # vim: ft=apparmor
 
-@{gs_file_ext}=[pP][dD][fF] [pP][sS] [eE][pP][sS] [eE][pP][sS][iI] [pP][nN][gG] [jJ][pP][gG] [jJ][pP][eE][gG] [pP][nN][mM] [tT][iI][fF] [tT][iI][fF][fF] [bB][mM][pP] [pP][cC][xX] [pP][sS][dD] [tT][xX][tT] [pP][xX][lL] [dD][oO][cC][xX] [xX][pP][sS]
+@{gs_file_ext}=[pP][dD][fF] [pP][sS] [eE][pP][sS] [eE][pP][sS][iI] [pP][nN][gG] [jJ][pP][gG] [jJ][pP][eE][gG] [pP][nN][mM] [tT][iI][fF] [tT][iI][fF][fF] [bB][mM][pP] [pP][cC][xX] [pP][sS][dD] [tT][xX][tT] [pP][xX][lL] [dD][oO][cC][xX] [xX][pP][sS] [xX][mM][lL] [iI][cC][cC] [rR][dD][fF]
 
 include if exists <tunables/gs.d>
